### PR TITLE
this is my change

### PR DESCRIPTION
**Pull request by mihaipotoroaca**
test 2

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
```
Comment (vladdraql) [inline .idea/vcs.xml]: ce-i asta??? b0$$

File: meHIGH.py [hunk 2]
```diff
@@ -0,0 +1 @@
+#ceva in el
```
Comment (vladdraql) [inline meHIGH.py]: pisica

File: .idea/vcs.xml (hunk 1 above)
Comment (cameliaenache) [reply .idea/vcs.xml]: meHigh Vapes rocks

Comment (mihaipotoroaca) [reply .idea/vcs.xml]: asa vreau eu! da ce-am facut safu? 🗡

Comment (miutesculazar) [reply .idea/vcs.xml]: Asta e Pomohaci

File: meHIGH.py (hunk 2 above)
Comment (mihaipotoroaca) [reply meHIGH.py]: cutzu! 💯

**Pull request merged**